Dockerfile.template

```diff
@@ -6,13 +6,12 @@ RUN groupadd --gid 1000 node \
 # gpg keys listed at https://github.com/nodejs/node#release-team
 RUN set -ex \
   && for key in \
-    9554F04D7259F04124DE6B476D5A82AC7E37093B \
     94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
     FD3A5288F042B6850C66B31F09FE44734EB7990E \
     71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
     DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
-    B9AE9905FFD7803F25714661B63B535A4C206CA9 \
     C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
+    B9AE9905FFD7803F25714661B63B535A4C206CA9 \
```

Comment:
This is just moving down -- is this just to get them into the same order as they're listed on https://github.com/nodejs/node#release-team? (not a maintainer, just trying to understand the logic behind the PR :+1:)

Reply:
Yes. I just copied the list on the release-team page as a block and added the \ on the end.

Reply:
This is largely cosmetic, but useful when comparing to the release page. As my previous comment, this change should also be made to the other `*.template` files